config/environments/production.rb

```diff
@@ -34,9 +34,7 @@
   config.assets.compile = false
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = proc do |_source, request|
-    Settings.find_assets_url if Settings.find_url.include?(request.host)
-  end
```

Comment:
Just for the record, the asset host in all environments is nil. This is a leftover from Gov.uk PaaS transition to AKS. So removing it to not confuse devs (like I got confused and wasting time).